src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { forwardRef, useRef } from "react";
+import cx from "classnames";
+import { useMergeRefs } from "../../hooks";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import { getTestId } from "../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import styles from "./Badge.module.scss";
+import { BadgeColor, BadgePosition, BadgeType } from "./BadgeConstants";
+import Indicator, { IndicatorProps } from "./Indicator/Indicator";
+import Counter, { CounterProps } from "../Counter/Counter";
+import { getStyle } from "../../helpers/typesciptCssModulesHelper";
+import { camelCase } from "lodash-es";
+import { IndicatorColor } from "./Indicator/IndicatorConstants";
+import { CounterColor } from "../Counter/CounterConstants";
+
+const BadgeColorToIndicatorColor = {
+  [BadgeColor.PRIMARY]: IndicatorColor.PRIMARY,
+  [BadgeColor.NOTIFICATION]: IndicatorColor.NOTIFICATION
+};
+
+const BadgeColorToCounterColor = {
+  [BadgeColor.PRIMARY]: CounterColor.PRIMARY,
+  [BadgeColor.NOTIFICATION]: CounterColor.NEGATIVE
+};
+
+export interface BadgeProps extends VibeComponentProps {
+  type?: BadgeType;
+  position?: BadgePosition;
+  color?: BadgeColor;
+  outbound?: boolean;
+  circular?: boolean;
+  border?: boolean;
```

Comment:
can you give me context why we add this?

Reply:
talked about it in the story meeting